File: src/main/java/com/ychstudio/ConvertKit.java
package com.ychstudio;


import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

public class ConvertKit extends Application {

    String version = "v0.1";

    public static void main(String[] args) {
        launch(args);
    }

    @Override
    public void start(Stage primaryStage) throws Exception {
        Parent fxmlRoot = FXMLLoader.load(getClass().getClassLoader().getResource("ConvertKitApplication.fxml"));

        Scene scene = new Scene(fxmlRoot);

        primaryStage.setScene(scene);
        primaryStage.setTitle("ConvertKit " + version);
        primaryStage.show();

    }
}

File: src/main/kotlin/com/ychstudio/ConvertKit.kt
package com.ychstudio

import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Parent
import javafx.scene.Scene
import javafx.stage.Stage

val version: String
    get() = "v0.1"

class MainWindow: Application() {
    override fun start(primaryStage: Stage) {

        val fxmlRoot = FXMLLoader.load<Parent>(javaClass.classLoader.getResource("ConvertKitApplication.fxml"))

        val scene = Scene(fxmlRoot)

        primaryStage.scene = scene
        primaryStage.title = "ConvertKit " + version
        primaryStage.show()
    }
}


fun main(args: Array<String>) {
    Application.launch(MainWindow::class.java, *args)
}